imap_processing/hit/l1b/hit_l1b.py

```diff
@@ -1,179 +1,60 @@
 """IMAP-HIT L1B data processing."""
 
 import logging
-from dataclasses import fields
 
-import numpy as np
 import xarray as xr
 
-from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
-from imap_processing.hit.l0.data_classes.housekeeping import Housekeeping
-from imap_processing.spice.time import met_to_j2000ns
+from imap_processing.hit.hit_utils import (
+    HitAPID,
+    get_attribute_manager,
+    get_datasets_by_apid,
+    process_housekeeping_data,
+)
 
 logger = logging.getLogger(__name__)
 
 # TODO review logging levels to use (debug vs. info)
 
 
-def hit_l1b(l1a_dataset: xr.Dataset, data_version: str) -> list[xr.Dataset]:
+def hit_l1b(dependencies: dict, data_version: str) -> list[xr.Dataset]:
     """
     Will process HIT data to L1B.
 
+    Processes dependencies needed to create L1B data products.
+
     Parameters
     ----------
-    l1a_dataset : xarray.Dataset
-        L1A data.
+    dependencies : dict
+        Dictionary of dependencies that are L1A xarray datasets
+        for science data and a file path string to a CCSDS file
+        for housekeeping data.
```

Comment:
I actually reread this comment and now I would like to suggest pulling this out into two variables, one for L1A xarrays, one for filepath strings, and maybe make them optional or something. Mixing them together into one dictionary seems dangerous...

Reply:
Yeah I don't love this either. It's a weird case because the housekeeping L1B product depends on the the raw ccsds packet file rather than an xarray dataset. I kind of want to see the ccsds packet reading code in cli.py since all instruments need it and that way the only data type getting passed into the different processing levels is xarray datasets. This would also reduce redundancy of the packet reading code being used in every instrument's L0 processing code. Something to discuss with the team